.github/workflows/codeql-analysis.yml

```diff
@@ -50,6 +50,9 @@ jobs:
       uses: github/codeql-action/init@v2
       with:
         languages: ${{ matrix.language }}
+        paths-ignore:
+          - 'bundler/spec/fixtures/projects/bundler1/invalid_ruby/Gemfile'
+          - 'bundler/spec/fixtures/projects/bundler2/invalid_ruby/Gemfile'
```

Comment:
Their docs show this goes in a separate config file that then gets configured in the workflow file, but I was hoping I could get away with putting it directly into the workflow file... no dice, at least not this way: https://github.com/dependabot/dependabot-core/actions/runs/4751969153

Reply:
Actually, looks like there's a PR close to being merged that'd let us pass config directly in the workflow file rather than the indirection of _yet-another-config-file_...
* https://github.com/github/codeql-action/pull/1590

Reply:
This should be unblocked now: https://github.com/github/codeql-action/blob/main/CHANGELOG.md#233---04-may-2023